packages/admin-ui/src/Accordion/components/AccordionContent.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
+import * as AccordionPrimitive from "@radix-ui/react-accordion";
+import { cva, type VariantProps, cn } from "~/utils";
+
+const accordionContentVariants = cva(
+    [
+        "wby-overflow-hidden wby-text-md",
+        "wby-transition-all data-[state=closed]:wby-animate-accordion-up data-[state=open]:wby-animate-accordion-down"
+    ],
+    {
+        // Using pixel values here because of non-existing design tokens.
+        variants: {
+            withIcon: {
+                true: "wby-pl-[36px]"
```

Comment:
Shall we use `wby-pl-5` and `wby-pl-9` instead? 
Generally, we don't have tokens for every possible spacing value, shall we use Tailwind classnames when possible?

I leave it as comment, not a blocker.